manifests/init.pp

```diff
@@ -167,8 +167,7 @@
       }
     }
     default: {
-      assert_type(Boolean, $selinux)
```

Comment:
This assert was redundant.   We've already confirmed the `Optional[Boolean]` variable wasn't `undef` and besides, `undef` is still falsey so the `mrepo::selinux` class wouldn't care if `$use_selinux` had been assigned `false` or `undef`